kong/plugins/httplog/schema.lua

```diff
@@ -0,0 +1,6 @@
+return {
+  logging_url = { required = true, type = "string" },
+  method = { require = false, default = "POST", enum = { "POST", "PUT", "PATCH" } },
```

Comment:
Do not enforce an enum, leave it free to be whatever the user wants.

Otherwise, everything looks much better :+1: